### DISCUSSION TITLE
📢 Tag creation on Stack Overflow

### DISCUSSION THREAD

**Discussion by maifeeulasad** [proyecto26/RestClient#210] [General]
## Hello good people 👋,
I couldn't contribute anymore after my first contribution as my finals are going on. But I thought it would be really cool to contribute somehow at least. I was exploring Stack Overflow. And [looked ](https://stackoverflow.com/search?q=proyecto26)for `proyecto26`, out of curiosity. And found some questions about us, as of now. So I was thinking about some tag creation on Stack Overflow 💡.

I've enough _(>1500)_ reputation on Stack Overflow to create a tag. So I thought it would be better to discuss this with all of you.  What should be the name of the tag:

- RestClient  ( ⚠️ there is a tag called `rest-client` for ruby)
- proyecto26-RestClient
- RestClient-proyecto26

Or if you guys have any other suggestions, please let me know. It would be really awesome, cause we solve together 💪.

I hope it would help us, to move all the Q/A, debugging type issues to Stack Overflow. On-demand(on bug on our end/faq), we can create an issue here. It may also help us to grow more.

Keep me in your prayers, till then stay well. Thanks. 🙏

**Comment by jdnichollsc** [comment]
Hello mate, sounds interesting!

I like the idea, what do you think about the name `Proyecto26.RestClient`? or if you prefer just `RestClient`, go ahead! <3

> It may also help us to grow more.

Absolutely, sounds perfect to be part of a great community!

Blessings,
Juan

**Reply by maifeeulasad** [reply]
Sounds awesome. 💌 

Let me know others' opinions then I'll create based upon that. I think we should wait till 26 March(1 week from now), at least.

**Reply by maifeeulasad** [reply]
@jdnichollsc it's 26 March, as none other than you and me are the only participant, I guess I'll create a tag with `proyecto26.restclient` (StackOverflow makes it). It's more descriptive, I hope it may help when we search with any of the keywords.

**Comment by maifeeulasad** [comment]
Okay, created tag named `proyecto26.restclient`. And also added as many related questions as possible.

Available here: https://stackoverflow.com/questions/tagged/proyecto26.restclient.

**Reply by jdnichollsc** [reply]
That's awesome, thanks for your help mate! <3
